makefile

```diff
@@ -29,12 +29,19 @@ build:
 	@ -brew cask install virtualbox
 	@ echo "\n${CHECK} Done"
 
-	@ echo "${HR}\nInstalling Ruby...${HR}\n"
-	@ -brew install ruby
+	@ echo "${HR}\nInstalling Ruby via rbenv...${HR}\n"
+	@ -brew install rbenv
+	@ -rbenv init
+	@ -rbenv install 2.3.0
+	@ -rbenv global 2.3.0
 	@ echo "\n${CHECK} Done"
 
+	# echo "${HR}\nInstalling Ruby...${HR}\n"
+	# -brew install ruby
+	# echo "\n${CHECK} Done"
```

Comment:
Shall we remove this commented code before it's merged, we can always get it from git if anything explodes